api/envoy/api/v2/cluster/filter.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package envoy.api.v2.cluster;
+
+option java_outer_classname = "FilterProto";
+option java_multiple_files = true;
+option java_package = "io.envoyproxy.envoy.api.v2.cluster";
+option csharp_namespace = "Envoy.Api.V2.ClusterNS";
+option ruby_package = "Envoy.Api.V2.ClusterNS";
+
+import "google/protobuf/any.proto";
+import "google/protobuf/struct.proto";
+
+import "validate/validate.proto";
+import "gogoproto/gogo.proto";
+
+option (gogoproto.equal_all) = true;
+
+// [#protodoc-title: Upstream filters]
```

Comment:
Can you add some arch docs somewhere in here? https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/upstream/upstream. Maybe link back and relate to https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/listeners/network_filters?

Reply:
OK, will add more documentation. Thanks!